src/lib/RepStatus.js

```diff
@@ -4,13 +4,18 @@ import { STATUS_SUCCESS, STATUS_VIEWABLE } from './constants';
 
 const STATUS_UPDATE_INTERVAL_MS = 2000;
 
+const ERROR_PASSWORD_PROTECTED = 'error_password_protected';
+const ERROR_TRY_AGAIN_LATER = 'error_try_again_later';
+const ERROR_UNSUPPORTED_FORMAT = 'error_unsupported_format';
+const ERROR_REFRESH = 'error_refesh';
+
 class RepStatus extends EventEmitter {
     /**
      * Gets the status out of represenation
      *
      * @public
      * @param {Object} representation - representation object
-     * @return {string} rep status instance
+     * @return {string} rep status state
      */
     static getStatus(representation) {
```

Comment:
Not sure this makes sense to be called getStatus() anymore since it's technically returning the state of the RepStatus rather than the status itself. Might just be a nitpick but thoughts?

Reply:
Agreed that it is a little odd but I think that the `state` is actually the `status` since the `status` as we knew it became an object a while back.